HandledError.py

```diff
@@ -0,0 +1,15 @@
+from flask import jsonify
+
+
+class HandledError(Exception):
```

Comment:
Why did you choose name Handled?

Reply:
Becouse errors will be handled :))

Reply:
I would that such classes are called like `AppError`, `CustomError`, {my-app-name}Error` more often,
e.g. there is `FlaskError`, `MongoError` and, to distinguish from them, your `AppError`

Reply:
fixed